### PR TITLE
Use fulltext indexing

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: lib/qa/authorities/getty.rb [hunk 1]
```diff
@@ -28,13 +28,14 @@ def build_query_url q
...
-                  FILTER langMatches( lang(?name), \"EN\" ) .
-                } LIMIT 10"
+      search = untaint(q)
+      # The full text index matches on fields besides the term, so we filter to ensure the match is in the term.
+      sparql = "SELECT ?s ?name {
+              ?s a skos:Concept; luc:term \"#{search}\";
+                 skos:inScheme <http://vocab.getty.edu/#{@sub_authority}/> ;
+                 gvp:prefLabelGVP [skosxl:literalForm ?name].
+              FILTER regex(?name, \"#{search}\", \"i\") .
+            } LIMIT 10"
```
Comment (awead) [inline lib/qa/authorities/getty.rb]: I would have thought you'd need skos and gvp prefixes defined, but if you're sure this query works for you, I'll merge ie.

**Comment by jeremyf** [comment]
Ping @awead

**Comment by awead** [comment]
@jcoyne @jeremyf if the sparql query is working for you, then :+1:

**Comment by jcoyne** [comment]
It's working fine on my server.

**Pull request merged**